align/compiler/write_verilog_lef.py

```diff
@@ -214,6 +214,69 @@ def generate_lef(name:str, attr:dict, available_block_lef:list, design_config:di
             unit_size_mos = design_config["unit_size_nmos"]
         else:
             unit_size_mos = design_config["unit_size_pmos"]
+
+        if unit_size_mos is None:
```

Comment:
Just want to make sure this condition applies only to align.pdk.finfet. Long term, the pdk (and its primitive generators) should control which parameters are required vs. not required.

Reply:
All other PDKs specify unit_size_* parameters.